common/archiver/s3store/historyArchiver_test.go

```diff
@@ -99,7 +99,7 @@ func (s *historyArchiverSuite) SetupTest() {
 	s.Assertions = require.New(s.T())
 	s.container = &archiver.HistoryBootstrapContainer{
 		Logger:        testlogger.New(s.T()),
-		MetricsClient: metrics.NewClient(scope, metrics.HistoryArchiverScope),
+		MetricsClient: metrics.NewClient(scope, metrics.History),
```

Comment:
this is a ServiceIdx, not a ScopeIdx.  as it's a test, I just updated it to something plausible.